src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -724,3 +724,55 @@ def test_wait_for_event_repeated_capabilities(session, channels):
     channels_session = session.channels[channels]
     with channels_session.initiate():
         channels_session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+def test_lcr_attributes(session):
+    session.lcr_open_compensation_enabled = True
+    session.lcr_short_compensation_enabled = True
+    session.lcr_load_compensation_enabled = True
+    session.lcr_open_short_load_compensation_data_source = (
+        nidcpower.LCROpenShortLoadCompensationDataSource.AS_DEFINED
+    )
+    session.lcr_open_conductance = 1.0
+    session.lcr_open_susceptance = 1.0
+    session.lcr_short_resistance = 1.0
+    session.lcr_short_reactance = 1.0
+    session.lcr_measured_load_resistance = 10.0
+    session.lcr_measured_load_reactance = 10.0
+    session.lcr_actual_load_resistance = 10.0
+    session.lcr_actual_load_reactance = 10.0
+    assert session.lcr_open_compensation_enabled
+    assert session.lcr_short_compensation_enabled
+    assert session.lcr_load_compensation_enabled
+    assert session.lcr_open_short_load_compensation_data_source == (
+        nidcpower.LCROpenShortLoadCompensationDataSource.AS_DEFINED
+    )
+    assert session.lcr_open_conductance == 1.0
+    assert session.lcr_open_susceptance == 1.0
+    assert session.lcr_short_resistance == 1.0
+    assert session.lcr_short_reactance == 1.0
+    assert session.lcr_measured_load_resistance == 10.0
+    assert session.lcr_measured_load_reactance == 10.0
+    assert session.lcr_actual_load_resistance == 10.0
+    assert session.lcr_actual_load_reactance == 10.0
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+@pytest.mark.parametrize(
+    "compensation_function",
```

Comment:
```suggestion
    "compensation_method",
```

The correct nomenclature for attributes and functions in Python is properties and methods.

Reply:
Thanks for the info, I've renamed it in the latest commit